vars/RedHat_8.yml

```diff
@@ -0,0 +1,28 @@
+---
+# PostgreSQL vars for RedHat 8+ based distributions
+#
+# Using a different cluster name could cause problems with SELinux.
+# See /usr/lib/systemd/system/postgresql-*.service
+postgresql_cluster_name: "data"
+postgresql_service_name: "postgresql-{{ postgresql_version }}"
+
+postgresql_varlib_directory_name: "pgsql"
+
+# Used to execute initdb
+postgresql_bin_directory: "/usr/pgsql-{{postgresql_version}}/bin"
+
+postgresql_unix_socket_directories:
+  - "{{ postgresql_pid_directory }}"
+  - /tmp
+
+postgresql_fdw_mysql_packages: "mysql_fdw_{{ postgresql_version_terse }}"
+postgresql_fdw_ogr_packages: "ogr_fdw{{ postgresql_version_terse }}"
+
+postgresql_packages:
+  - ca-certificates
+  - python3-psycopg2
+  - python3-pycurl
+  - glibc-common
+  - epel-release
```

Comment:
Please verify if EPEL is required for RockyLinux 8,9.
Maybe, it is not required.

Reply:
It was not required, I have removed from the list